ci-droid-tasks-consumer-services/src/main/java/com/societegenerale/cidroid/tasks/consumer/services/eventhandlers/BestPracticeNotifierHandler.java

```diff
@@ -27,77 +28,112 @@
 
     private ResourceFetcher resourceFetcher;
 
+    private int maxFilesInPr;
+
+    private String maxFilesWarningMessage;
+
     public BestPracticeNotifierHandler(Map<String, String> configuredPatternToContentMapping,
-                                       List<Notifier> notifiers, RemoteGitHub remoteGitHub, ResourceFetcher resourceFetcher) {
+                                       List<Notifier> notifiers, RemoteGitHub remoteGitHub, ResourceFetcher resourceFetcher,
```

Comment:
while I agree that limiting the size of PRs is  a best practice, I feel this should be a dedicated PullRequestEventHandler. Can you move this to a dedicated class like PullRequestSizeCheck please ? this will help keep the logic separated

Reply:
done.